src/transaction/log_replication.cpp

```diff
@@ -438,12 +439,28 @@ namespace cublog
     return m_most_recent_trantable_snapshot_lsa.load ();
   }
 
-  log_lsa replicator::get_redo_lsa () const
+  log_lsa replicator::get_highest_processed_lsa () const
   {
     std::lock_guard<std::mutex> lockg (m_redo_lsa_mutex);
     return m_redo_lsa;
   }
 
+  log_lsa replicator::get_lowest_unapplied_lsa () const
+  {
+    // TODO: needs to be refactored to work with the new replicators flavors
+    // for now it should not be called
+    assert (false);
+
+    if (m_parallel_replication_redo == nullptr)
+      {
+	//sync
+	std::lock_guard<std::mutex> lockg (m_redo_lsa_mutex);
+	return m_redo_lsa;
+      }
+    //async
+    return m_parallel_replication_redo->get_min_unapplied_log_lsa ();
```

Comment:
when atomic replication will be implemted; this function will return the value of atomic replication "minimum unapplied lsa"; therefore `m_parallel_replication_redo->get_min_unapplied_log_lsa` is not needed